utils/debug/MaterialUV.js

```diff
@@ -0,0 +1,14 @@
+import * as THREE from 'three';
+import materialUV_VS from './shader/debugUV_VS.glsl';
+import materialUV_FS from './shader/debugUV_FS.glsl';
+
+const materialUV = new THREE.ShaderMaterial({
```

Comment:
I don't think we need to merge this in iTowns: this is some simple debug code with nothing itowns specific in it.